src/constants.lua

```diff
@@ -1,13 +1,19 @@
+--[[
+	NOTE: constants is used throughout the codebase, so avoid imports of any modules in this file to prevent circular dependencies
+]]
+--
 local constants = {}
 
+--- CONVERSION HELPERS
+constants.DENOMINATION = 6
 -- @alias mARIO number
 -- intentionally not exposed so all callers use ARIOToMARIO for consistency
-local mARIO_PER_ARIO = 1000000
+local mARIO_PER_ARIO = 10 ^ constants.DENOMINATION -- 1 million mARIO per ARIO
```

Comment:
The docs do say that exponentiation should work this way but a skeptical part of me almost wants a unit test assertion to make sure this constant's value is 1000000 😬

Reply:
tests added [86840f3](https://github.com/ar-io/ar-io-network-process/pull/363/commits/86840f35004fd73247d5ddeea5106dcd3a4dab26)